scripts/run_train.py

```diff
@@ -452,6 +465,19 @@ def main() -> None:
     logging.info(f"Number of parameters: {tools.count_parameters(model)}")
     logging.info(f"Optimizer: {optimizer}")
 
+    if args.wandb:
+        logging.info("Using Weights and Biases for logging")
+        wandb_config = {}
+        args_dict = vars(args)
```

Comment:
maybe save this args dict as a json?

Reply:
what is the use case of that?

Reply:
I have added it to the wandn summary dict. Basically it is just easier to parse compared to the one-liner at the top of the log file, but contains the same information.

Reply:
I see, we could also make this one liner a bit more readable. it get saves to the wandb forlder?

Reply:
yes, it is in the `wandb-summary.json` file under the keyword `params`